robottelo/api/utils.py

```diff
@@ -63,7 +67,17 @@ def enable_rhrepo_and_fetchid(basearch, org_id, product, repo, reposet, releasev
     if releasever is not None:
         payload['releasever'] = releasever
     payload['product_id'] = product.id
-    r_set.enable(data=payload)
+    try:
+        r_set.enable(data=payload)
+    except HTTPError as e:
+        if (
```

Comment:
Functionally this is fine, but consider organizing for shortest 'path' through the block for readability, and clear indication of what `strict` will do.  Since this is a backport I can understand wanting to keep it consistent with the PR that's already been merged.

```
if strict:
    # strict is set, the other checks don't matter, raise
    raise
# we're not in strict mode, now consider the other checks
else if e.response.status_code == 409 and 'repository is already enabled' in e.response.json()['displayMessage']:
    pass
```